src/services/refactors/extractMethod.ts

```diff
@@ -932,7 +933,8 @@ namespace ts.refactor.extractMethod {
         scopes: Scope[],
         enclosingTextRange: TextRange,
         sourceFile: SourceFile,
-        checker: TypeChecker) {
+        checker: TypeChecker,
+        cancellationToken: CancellationToken) {

```

Comment:
With the other changes, I don't think this argument is used anywhere anymore.

Reply:
If you mean `cancellationToken`, I believe it is closed over in the accept lambda below.

Reply:
@amcasey Ack, sorry, thought I was reading a different file here. Nevermind. All good.